StripeFinancialConnections/StripeFinancialConnections/Source/Native/NetworkingLinkLoginWarmup/NetworkingLinkLoginWarmupViewController.swift

```diff
@@ -87,6 +87,13 @@ final class NetworkingLinkLoginWarmupViewController: UIViewController {
                         didSelectSkipWithManifest: manifest
                     )
                 case .failure(let error):
+                    self.dataSource
```

Comment:
no need for `success`?

Reply:
What do you mean by that? Note that this particular DIFF clips out the success case above (success case can be seen by extending the code)

Reply:
ah ok, I thought I saw something like `if case .failure(let error) = result`